src/components/sidesheet/sidesheet.tsx

```diff
@@ -24,11 +24,11 @@ const Sidesheet = ({
         <Transition.Child
           as={Fragment}
           enter="ease-out duration-300"
-          enterFrom="opacity-0"
-          enterTo="opacity-100"
+          enterFrom="opacity-0 z-0"
```

Comment:
I also checked the dialog if this clashes but the dialog is implemented as a portal, so the dialog should be always on top.

Reply:
I see, do we leave it like this to make it look good on Storybook or should I remove it?